digi/xbee/reader.py

```diff
@@ -686,10 +723,101 @@ def del_micropython_data_received_callback(self, callback):
             callback (Function): the callback to delete.
 
         Raises:
-            ValueError: if ``callback`` is not in the callback list of :class:`.MicroPythonDataReceived` event.
+            ValueError: if ``callback`` is not in the callback list of
+                :class:`.MicroPythonDataReceived` event.
         """
         self.__micropython_data_received -= callback
 
+    def get_packet_received_callbacks(self):
+        """
+        Returns the list of registered callbacks for received packets.
+
+        Returns:
+            List: List of :class:`.PacketReceived`
```

Comment:
NP: missing 'event.' an the end of the line.

Reply:
Also, shouldn't it be 'events' instead of 'event'?

Reply:
Done.